src/Symfony/Component/AssetMapper/ImportMap/ImportMapConfigReader.php

```diff
@@ -127,11 +133,11 @@ public function findRootImportMapEntry(string $moduleName): ?ImportMapEntry
         return $entries->has($moduleName) ? $entries->get($moduleName) : null;
     }
 
-    public function createRemoteEntry(string $importName, ImportMapType $type, string $version, string $packageModuleSpecifier, bool $isEntrypoint): ImportMapEntry
+    public function createRemoteEntry(string $importName, ImportMapType $type, string $version, string $packageModuleSpecifier, bool $isEntrypoint, ?string $resolver): ImportMapEntry
```

Comment:
adding an argument is a BC break so we need to either find another approach or do a proper BC layer (using `@param` + func_get_arg, see branch 6.4 for inspiration)